libraries/MySensors/MyConfig.h

```diff
@@ -101,7 +101,7 @@
 #define MY_ATSHA204_PIN 17 // A3 - pin where ATSHA204 is attached
 
 // MySigningAtsha204Soft default settings
-#define MY_RANDOMSEED_PIN 7 // A7 - Pin used for random generation (do not connect anything to this)
+#define MY_RANDOMSEED_PIN A0 // A0 - Pin used for random generation (do not connect anything to this)
```

Comment:
The random generator used by the software SHA204 implementation claims the only analog input pin of the ESP. Can we make use of a different random generator for the ESP implementation?